skins/cat17/templates/page_layouts/use_of_funds.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends 'page_layouts/base.html.twig' %}
+{% set page_works_without_js = false %}
+
+{% block main_content %}
+    <div id="funds" data-content="{$ use_of_funds_content $}"></div>
```

Comment:
Maybe add a `<noscript>` section with links to the wmde and wmf homepage where that can be found? Or at least a notice of "you need to enable javascript to read this."